src/scripts/commands/metadata/sync.ts

```diff
@@ -47,27 +43,49 @@ export const syncMetadata = command({
             description: "Path to csv file with DHIS2 models to ignore (optional)",
             defaultValue: () => "",
         }),
+        action: option({
+            type: string,
+            long: "action",
+            description: "Action to perform (CREATE | CREATE_AND_UPDATE | DELETE)",
+            defaultValue: () => "",
+        }),
+        persist: flag({
+            type: boolean,
+            long: "persist",
+            description: "Persist changes to the server (true/false)",
+            defaultValue: () => false,
+        }),
```

Comment:
Typically, flag is used like this:


persist: flag({
            long: "persist",
            description: "Persist changes to the server",
        }),

As flag implies boolean with a default false